src/navigation/ActivationStack.tsx

```diff
@@ -121,6 +122,13 @@ const ActivationStack: FunctionComponent = () => {
 
   return (
     <Stack.Navigator screenOptions={screenOptions}>
+      <Stack.Screen
```

Comment:
Should this screen be part of the activation steps or am I missing something?

Reply:
E.g.

```
const acceptEula: ActivationStep = {
  screenName: ActivationScreens.AcceptEula,
  component: AcceptEula,
}

const activationStepsAndroid: ActivationStep[] = isLocationNeeded
  ? [acceptEula, activateProximityTracing, activateLocation]
  : [acceptEula, activateProximityTracing]

const activationStepsIOS: ActivationStep[] = [
  acceptEula,
  activateProximityTracing,
  notificationPermissions,
]
```

Reply:
Good catch, It probably is. This will update the right side with step x out of y right? In this PR I'm gonna remove this part since we are not activating the accept eula screen just yet, but I'll open a new one with this next. 👍

Reply:
Yes! Cool cool.